### PR TITLE
Adding Android SDK to Ubuntu 16.04

### PR DESCRIPTION
**Pull request by joshmgross**
Adding the Android SDK and build tools to Ubuntu 16.04 for parity with Windows VS2017 agent

### CODE REVIEW

File: images/linux/scripts/installers/android.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    "add-ons;addon-google_apis-google-24" \
+    "add-ons;addon-google_apis-google-23" \
+    "add-ons;addon-google_apis-google-22" \
+    "add-ons;addon-google_apis-google-21" \
+    "cmake;3.6.4111459" \
+    "patcher;v4"
+
+# Document what was added to the image
+echo "Lastly, documenting what we added to the metadata file"
+DocumentInstalledItem "Android SDK [${ANDROID_ROOT}]"
```
Comment (chrisrpatterson) [inline images/linux/scripts/installers/android.sh]: We should document all of the installed versions like https://github.com/Microsoft/vsts-image-generation/blob/master/images/win/scripts/Installers/Update-AndroidSDK.ps1#L124

File: images/linux/scripts/installers/android.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+source $HELPER_SCRIPTS/apt.sh
+
+# Set env variable for SDK Root (https://developer.android.com/studio/command-line/variables)
+ANDROID_ROOT=/usr/local/bin/android
+ANDROID_ROOT_SDK=${ANDROID_ROOT}/sdk
+echo "ANDROID_ROOT_SDK=${ANDROID_ROOT_SDK}" | tee -a /etc/environment
```
Comment (chrisrpatterson) [inline images/linux/scripts/installers/android.sh]: Should this be ANDROID_HOME

https://github.com/Microsoft/vsts-image-generation/blob/master/images/win/scripts/Installers/Update-AndroidSDK.ps1#L124

Comment (joshmgross) [reply images/linux/scripts/installers/android.sh]: ANDROID_HOME is deprecated, see: https://developer.android.com/studio/command-line/variables

File: images/linux/scripts/installers/android.sh (hunk 1 above)
Comment (joshmgross) [reply images/linux/scripts/installers/android.sh]: I was worried about cluttering up https://github.com/Microsoft/vsts-image-generation/blob/master/images/linux/Ubuntu1604-README.md by adding every specific version

Comment (chrisrpatterson) [reply images/linux/scripts/installers/android.sh]: People wan to know that level of detail.  It is important.

Comment (dakale) [reply images/linux/scripts/installers/android.sh]: We typically document every version even if theres multiples, see go, java, php etc

File: images/linux/scripts/installers/android.sh (hunk 2 above)
Comment (chrisrpatterson) [reply images/linux/scripts/installers/android.sh]: ok then we should fix the other images @lokesh755

Comment (lokesh755) [reply images/linux/scripts/installers/android.sh]: Sure!

**Pull request merged**